doc/reference/configuration/configuration_reference.rst

```diff
@@ -11,6 +11,329 @@ This topic describes all :ref:`configuration parameters <configuration>` provide
 Most of the configuration options described in this reference can be applied to a specific instance, replica set, group, or to all instances globally.
 To do so, you need to define the required option at the :ref:`specified level <configuration_scopes>`.
 
+..  _configuration_reference_audit:
+
+audit_log
+---------
```

Comment:
Shouldn't we keep the `config` section first?

Reply:
Since it is a reference, it seems logical to keep the alphabetical order.

See also [another comment](https://github.com/tarantool/doc/pull/3983#discussion_r1480284197) on the topic.